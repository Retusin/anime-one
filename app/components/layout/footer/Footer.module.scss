.footer {
  @apply bg-[#343a40];

  &_wrapper {
    @apply mx-auto max-w-[1310px] px-[15px] py-2 flex justify-between text-[#999999];
  }

  &_list {
    @apply flex gap-x-10;
  }
}
