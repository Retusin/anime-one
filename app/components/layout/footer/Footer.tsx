import { FC } from 'react';

import styles from './Footer.module.scss';

export const Footer: FC = () => (
  <div className={styles.footer}>
    <div className={styles.footer_wrapper}>
      <div className={styles.footer_list}>
        <span>Accord</span>
        <span>Privacy</span>
        <span>For copyright holders</span>
      </div>
      <div>Â© RetusinðŸ’™ 2022</div>
    </div>
  </div>
);
