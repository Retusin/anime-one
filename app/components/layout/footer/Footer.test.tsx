import { Footer } from '@/components/layout/footer/Footer';
import { render } from '@testing-library/react';

test('render footer', () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
