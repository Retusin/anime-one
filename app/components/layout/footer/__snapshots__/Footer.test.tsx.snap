// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render footer 1`] = `
<div>
  <div
    class="footer"
  >
    <div
      class="footer_wrapper"
    >
      <div
        class="footer_list"
      >
        <span>
          Accord
        </span>
        <span>
          Privacy
        </span>
        <span>
          For copyright holders
        </span>
      </div>
      <div>
        Â© RetusinðŸ’™ 2022
      </div>
    </div>
  </div>
</div>
`;
